src/Features/Core/Portable/Diagnostics/Analyzers/NamingStyles/NamingStylePreferencesInfo.cs

```diff
@@ -49,6 +40,12 @@ private bool IsSymbolNameAnalyzable(ISymbol symbol)
                 return false;
             }
 
+            var propertySymbol = symbol as IPropertySymbol;
```

Comment:
If this runs on every symbol we should attempt .SymbolKind == checks first.  'is/as' are expensive and do add up.